Updated Example Project

magorlick opened the pull request:
closes #481

AdityaHegde commented on scripts/example-project.sh:
```diff
@@ -8,11 +8,34 @@ echo "Initializing the project example project ${PROJECT_BASE}/rill-developer-ex
 
-echo "Importing example dataset into the project..."
-npm run cli --silent -- import-source ${PROJECT_BASE}/rill-developer-example/data/flightlist_2022_02.csv --project ${PROJECT_BASE}/rill-developer-example
+echo "Importing example datasets into the project..."
```
In order to make this work on Windows we are not using this `.sh` script anymore. You will have to update `src/cli/ExampleProjectCommand.ts`. We might have to use alternatives to `rm` and `mv` commands also.

magorlick replied:
Thanks @AdityaHegde , I did note that in the associated issue here. https://github.com/rilldata/rill-developer/issues/481#issuecomment-1179310653.  I am not sure I am the right person to finish the work, but I need to find the someone to help and I am not sure it will make it this sprint given our priorities.

ericpgreen2 replied:
I ported `scripts/example-project.sh` into `src/cli/ExampleProjectCommand.ts`, so I'll mark this as resolved.

AdityaHegde commented on src/cli/ExampleProjectCommand.ts:
```diff
@@ -28,24 +28,108 @@ export class ExampleProjectCommand extends DataModelerCliCommand {
+    );
+
+    console.log("Importing example SQL transformations into the project...");
+    execSync(`mv -v ${project}/example-assets/models/* ${project}/models`, {
```
Replace these mv and rm with nodejs APIs ? That way it will work on windows more easily (we still need an alternative to unzip). @magorlick If not we should mention that our app doesnt work on windows just yet.

magorlick replied:
This is a good call out. We will leave this as is for the next release and follow up when we tackle https://github.com/rilldata/rill-developer/issues/510.   I will make a user note in our docs that we are Mac and Linux only.

magorlick commented:
I made a few changes to the CLI to shorten the command and removed some of the examples.  I also removed the example from the downloaded zipped asset, and I am waiting for @himadrisingh to finish the upload.  Once he does the `curl` command should work.

magorlick commented:
> I made a few changes to the CLI to shorten the command and removed some of the examples. I also removed the example from the downloaded zipped asset, and I am waiting for @himadrisingh to finish the upload. Once he does the `curl` command should work.

Assets are ready.
